app/models/casa_case.rb

```diff
@@ -53,6 +53,7 @@ class CasaCase < ApplicationRecord
     joins(:case_assignments)
       .where(case_assignments: {active: true})
       .where.not(case_assignments: {volunteer: volunteer})
+      .where(casa_org: volunteer.casa_org)
```

Comment:
This is great! Please remove the other changes (javascript and package.json) and add a test for this so we don't break it again :)

Reply:
sorry I missed the message, although the code was merged, I will add the test when I have time, thanks for reviewing.